conf/core/ConfigOptions.asciidoc

```diff
@@ -1310,6 +1339,14 @@ The Linux utility RLIMIT_AS is used for limiting virtual memory. This is analago
 less RAM will be utilized than the value specified. Typically this is most useful when limiting
 RAM usage of applications in a shared server environment.
 
+=== maximal.subline.spacing
```

Comment:
This was part of my attempt to use maximal subline matching in addition to Frechet, which didn't pan out.  By raising this value you can increase the runtime performance of maximal quite a bit, but the algorithm gets unstable for some datasets when its raised too much.